### ISSUE TITLE
link stylesheet and script javascript

### ISSUE THREAD

**Issue by hueitan** [astoilkov/OneScript#2]
`<link rel="stylesheet" href="xxx.css"/>`

`<script type="text/javascript" src="xx.js"></script>`

**Comment by astoilkov** [comment]
Can you give a little more context? What do you mean by that?

**Comment by hueitan** [comment]
I didn't see anything about this in your example.

**Comment by astoilkov** [comment]
Yes because it stays the same. You could reference a external JavaScript and CSS the same way you will do it in regular page. I am only thinking in a shorthand syntax like this one (but not sure about it):

``` html
<css href="xxx.css" />
```

**Comment by hueitan** [comment]
:+1: cool for that.

**Comment by astoilkov** [comment]
From the little that could be seen - what do you think about OneScript?

**Comment by hueitan** [comment]
I have the same idea about how to writing a "component" means html/css/js should be in one file.

In traditional way, every times we need to find out the method, we may need to go though all the folders and files to get the method function.

In this kind of "component", we can directly find out the style and script. It's easier for maintaining and debugging.

I have thinking about this before and I found `webcomponent`it's quite useful. After that `reactjs`, seems everything writing is javascript within one file, but in the end it's not.

Actually it's quite surprise and happy that I found your repo and then `OneScript`, that's the one I always think about. But i have no idea what structure it should be, it's still be the imagination on our mind.

Cool, I love the project you have done. :beers:

**Comment by hueitan** [comment]
Hi @astoilkov What do you think about [polymer](https://www.polymer-project.org/1.0/)

**Comment by astoilkov** [comment]
Polymer is a great project. I love web components but it is too big a framework for me. I have been experimenting for the past couple of weeks and I believe the solution should be more complex for compiling and super simple for the client-side code. I am experimenting with something around the 500 lines of code.
